### ISSUE TITLE
Twilight Highlands completely no npc´s

### ISSUE THREAD

**Issue by LeRutY** [ottconnect/Legion#120]
No mob´s and no npc´s (at the flightpoints) in the zone.

**Comment by ottconnect** [comment]
do you have the cords for that location

**Comment by LeRutY** [comment]
Eastern Kingdoms, 55,8 / 53,8
http://www.wowhead.com/zone=4922

**Comment by ottconnect** [comment]
Excellent I'll review it once I get home.

**Comment by ottconnect** [comment]
yeah i see what you mean not even the quest giver is posted there, i added one quest giver today will add the rest and then the rest of the missing spawns.

i know a lot of TBC and Cata spawns are missing.
